Libraries/LibWeb/IndexedDB/IDBObjectStore.h

```diff
@@ -44,6 +53,9 @@ class IDBObjectStore : public Bindings::PlatformObject {
 
     // An object store handle has a name, which is initialized to the name of the associated object store when the object store handle is created.
     String m_name;
+
+    // An object store handle has an index set
+    Vector<GC::Ref<Index>> m_indexes;
```

Comment:
"set" normally implies AK::HashTable to us. Do the expected access/addtion/removal patterns here lean themselves more to a vector?